--- a/erts/emulator/Makefile.in	2018-03-22 01:33:28.000000000 +0800
+++ b/erts/emulator/Makefile.in	2018-04-18 21:32:51.853832265 +0800
@@ -705,7 +705,7 @@
 endif
 
 $(OBJDIR)/%.o: beam/%.c
-	$(V_CC) $(subst -O2, $(GEN_OPT_FLGS), $(CFLAGS)) $(INCLUDES) -c $< -o $@
+	$(V_CC) $(INCLUDES) $(subst -O2, $(GEN_OPT_FLGS), $(CFLAGS)) -c $< -o $@
 
 $(OBJDIR)/%.o: $(TARGET)/%.c
 	$(V_CC) $(CFLAGS) $(INCLUDES) -Idrivers/common -c $< -o $@
